Compile fails when the number of jobs is large

`make -j 22` at the top-level on a machine with a large number of cores

Associated compilation output:
```
cd .libs && ln -s -f pkcs11.so libpkcs11.so
/bin/bash: line 0: cd: .libs: No such file or directory
make[2]: *** [all-local] Error 1
make[2]: *** Waiting for unfinished jobs....
```
(Full log: http://errors.yoctoproject.org/Errors/Details/75462/)

It appears that the target `all-local` does not have the required dependencies defined explicitly

I added `all-local: $(LTLIBRARIES)` as a quick work around.  I can create a PR if that is a valid long term fix

--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -37,7 +37,7 @@ pkcs11_la_LDFLAGS = $(AM_LDFLAGS) -module -shared -shrext $(SHARED_EXT) \
 	-avoid-version -export-symbols "$(srcdir)/pkcs11.exports"
 
 # OpenSSL older than 1.1.0 expected libpkcs11.so instead of pkcs11.so
-check-local:
+check-local: $(LTLIBRARIES)
 	cd .libs && $(LN_S) -f pkcs11$(SHARED_EXT) libpkcs11$(SHARED_EXT)
 install-exec-hook:
 	cd '$(DESTDIR)$(enginesdir)' && $(LN_S) -f pkcs11$(SHARED_EXT) libpkcs11$(SHARED_EXT)

